### PR TITLE
DBZ-3353 Correct event.deserialization.failure.handling.mode documentation

### PR DESCRIPTION
**Pull request by Naros**
https://issues.redhat.com/browse/DBZ-3353

### CODE REVIEW

File: documentation/modules/ROOT/pages/connectors/mysql.adoc [hunk 1]
```diff
@@ -2229,15 +2229,15 @@ If you set this option to `true` then you must also configure MySQL with the `bi
  +
 Setting `include.query`to `true` might expose tables or fields explicitly excluded or masked by including the original SQL statement in the change event. For this reason, the default setting is `false`.
```
Comment (roldanbob) [inline documentation/modules/ROOT/pages/connectors/mysql.adoc:2230]: "... expose tables or fields that are explicitly excluded ..."

**Comment by roldanbob** [comment]
Thanks for the heads up, @Naros. Next week I'll be working on other updates to the file, so it'll be good to get this merged in. Is this applicable to 1.5 only or will we backport it to 1.4 as well? Might make a difference in how I handle the other updates. I was planning to apply the other updates to the 1.4 product doc, but I can always apply them downstream separately.

**Comment by Naros** [comment]
I raised the same concern in the Jira, I'd say lets discuss there and converge on a plan for how we want to address this for older versions of the documentation for both upstream & downstream.

File: documentation/modules/ROOT/pages/connectors/mysql.adoc (hunk 1 above)
Comment (jpechane) [inline documentation/modules/ROOT/pages/connectors/mysql.adoc:2230]:
```suggestion
Setting `include.query`to `true` might expose tables or fields that are explicitly excluded or masked by including the original SQL statement in the change event. For this reason, the default setting is `false`.
```

**Comment by jpechane** [comment]
@Naros Thanks, let's continue discussion in Jira as proposed. Still we should backport at least to `1.4` branch

**Pull request merged**